.npmrc

```diff
@@ -0,0 +1,2 @@
+save-exact = true
+progress = false
```

Comment:
i have to install yarn...

Reply:
This project uses https://volta.sh/ to manage this, sorry for the confusion!